src/components/WPScan/WPScan.tsx

```diff
@@ -120,285 +111,258 @@ const WPScan = () => {
+                                        />
+                                        <Switch
+                                            label="Force SSL"
+                                            {...form.getInputProps("forceSsl", { type: "checkbox" })}
+                                        />
+                                        <Switch
+                                            label="Disable TLS Checks"
+                                            {...form.getInputProps("disableTlsChecks", { type: "checkbox" })}
+                                        />
+                                        <TextInput label="HTTP Authentication" {...form.getInputProps("httpAuth")} />
+                                        <TextInput label="Proxy" {...form.getInputProps("proxy")} />
+                                    </Stack>
+                                )}
 
-                    <Button type={"submit"}>Scan</Button>
-                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                    <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-                </Stack>
-            </form>
-        </RenderComponent>
+                                {authOpened && (
+                                    <Stack mt={10}>
+                                        <TextInput label="Password List" {...form.getInputProps("passwords")} />
+                                        <TextInput label="Username List" {...form.getInputProps("usernames")} />
+                                        <TextInput label="API Token" {...form.getInputProps("apiToken")} />
+                                    </Stack>
+                                )}
+
+                                {outputOpened && (
+                                    <Stack mt={10}>
+                                        <Select
+                                            label="Output Format"
+                                            data={[
+                                                { value: "cli", label: "CLI" },
+                                                { value: "json", label: "JSON" },
+                                                { value: "cli-no-color", label: "CLI (No Color)" },
+                                            ]}
+                                            {...form.getInputProps("outputFormat")}
+                                        />
+                                        <TextInput label="Output File" {...form.getInputProps("outputFile")} />
+                                    </Stack>
+                                )}
+                            </Stepper.Step>
+                            <Stepper.Step label="Run">
+                                <Stack align="center" mt={20}>
+                                    <Button type="submit" disabled={loading} style={{ alignSelf: "center" }}>
+                                        Run WPScan
```

Comment:
The convention is Start + title of tool. 
```suggestion
                                        Start {title}
```